services/movie_session.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall_id=cinema_hall_id,
+                                movie_id=movie_id)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    session = MovieSession.objects.all()
+    if session_date:
+        return session.filter(show_time__date=session_date)
+    return session
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
```

Comment:
The function `get_movie_session_by_id` is expected to return a MovieSession object, but the `.get()` method in line 22 returns a QuerySet. Please correct the function return type to match the actual returned data type.